packages/eslint-config/index.cjs

```diff
@@ -68,4 +69,12 @@ module.exports = {
     // warnings so we can write appropriate interfaces and guards later.
     '@typescript-eslint/consistent-type-assertions': ['warn', { assertionStyle: 'never' }],
   },
+  overrides: [
+    {
+      files: ['apps/web/**/*.{tsx,ts}'],
+      rules: {
+        'no-process-env': 'error',
```

Comment:
restrict accessing `process.env` only for webapp

Reply:
ooo